test/integration_tests/test_pytorch.py

```diff
@@ -243,6 +243,27 @@ def test_should_run_ocr_on_cropped_data(self):
         self.assertTrue(res["ocrextractor.labels"][0][0] == "4")
         self.assertTrue(res["ocrextractor.scores"][2][0] > 0.9)
 
+    @pytest.mark.torchtest
+    def test_should_run_extract_object(self):
+        select_query = """
+            SELECT id, T.iids, T.bboxes, T.scores, T.labels
+            FROM MyVideo JOIN LATERAL EXTRACT_OBJECT(data, Yolo, NorFairTracker)
+                AS T(iids, labels, bboxes, scores)
+            WHERE id < 30;
+            """
+        actual_batch = execute_query_fetch_all(select_query)
+        self.assertEqual(len(actual_batch), 30)
+
+        select_query = """
+            SELECT id, T.iid, T.bbox, T.score, T.label
+            FROM MyVideo JOIN LATERAL
+                UNNEST(EXTRACT_OBJECT(data, Yolo, NorFairTracker)) AS T(iid, label, bbox, score)
+            WHERE id < 30;
+            """
+        actual_batch = execute_query_fetch_all(select_query)
+        # do some more meaningful check
+        self.assertEqual(len(actual_batch), 685)
```

Comment:
@gaurav274 Where does the number 685 come from? Does it not change with the version of yolo and tracker installed from pip?

Reply:
It might change. I wasn't sure about the check I should add.

Reply:
With the current Yolo. it is 625 instead of 685. I think this type test cases will be flaky. I suggest we either remove it, or use dummy detector and tracker, so the  results are deterministic.

Reply:
Actually, there is a better idea, we can get the count from the previous query.